src/views/Notes/NoteEditPage.js

```diff
@@ -18,19 +18,7 @@ export default class NoteEditRoute extends Component {
   };
 
   goToNoteView = () => {
-    const {
-      match,
-      history,
-      location,
-    } = this.props;
-
-    const { id } = match.params;
-    const noteViewUrl = `/users/notes/${id}`;
-
-    history.replace({
-      pathname: noteViewUrl,
-      state: location.state,
-    });
+    this.props.history.goBack();
```

Comment:
Don't you think would be better to rename the function according to the updated logic?